standard/unsafe-code.md

```diff
@@ -544,6 +544,8 @@ addressof_expression
     ;
 ```
 
+<!-- [Placeholder] 23.6.5 -->
+<!-- [Placeholder] 23.6.5 End -->
```

Comment:
### [Placeholder] 23.6.5

#### First Alternative

A certain compiler produces a compile time error in the following situation.

---
The *unary_expression* operand of an address-of operator may not be a *null_forgiving_expression*.

---

#### Second Alternative

---
Remove placeholder

---

Reply:
I'm fine with the first alternative here.

Reply:
```cs
#nullable enable

unsafe
{
    int e = 4;
    _ = &e!; // Error
    _ = &(e!); // Error
    _ = (&e)!; // No error
}
```

Reply:
(Just for the record: I've been completely convinced by removing this, i.e. alternative 2)